libsolidity/analysis/TypeChecker.cpp

```diff
@@ -2656,6 +2656,9 @@ bool TypeChecker::visit(MemberAccess const& _memberAccess)
 				_memberAccess.location(),
 				"Using \"." + memberName + "(...)\" is deprecated. Use \"{" + memberName + ": ...}\" instead."
 			);
+
+		if (!funType->bound())
```

Comment:
And if it is bound, the field is not set at all?

Reply:
Well, then it's the default, which is `false`. I could be more explicit I suppose and set it if bound to `false`

Reply:
The `optional`/`SetOnce` PR is a different one ;)